### PR TITLE
feat: support external pool `50wstETH-50ACX` for staking

### PR DESCRIPTION
**Pull request by dohaki**
Adds support for staking external Balancer pool `50wstETH-50ACX`.

Closes ACX-1351

### CODE REVIEW

**Review by james-a-morris — COMMENTED** [review]
Left a few nits.

File: src/views/Staking/components/StakingInputBlock/StakingInputBlock.tsx [hunk 1]
```diff
@@ -22,6 +24,7 @@ interface Props {
   invalid: boolean;
   buttonText: string;
   logoURI: string;
+  logoURIs?: [string, string];
```
[the remark this answers is not in the document]
Comment (dohaki) [reply src/views/Staking/components/StakingInputBlock/StakingInputBlock.tsx]: Changed https://github.com/across-protocol/frontend-v2/pull/798/commits/b9f3bbdd04a675a43775aeea679e0b9c0789c11d

File: src/views/Staking/types.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ export type StakingRewardPropType = GenericStakingComponentProps & {
 
 export type StakingFormPropType = GenericStakingComponentProps & {
   logoURI: string;
+  logoURIs?: [string, string];
```
[the remark this answers is not in the document]
Comment (dohaki) [reply src/views/Staking/types.ts]: I think your suggestion with the common type addresses this https://github.com/across-protocol/frontend-v2/pull/798/commits/b9f3bbdd04a675a43775aeea679e0b9c0789c11d

File: src/views/Staking/hooks/useStakeFormLogic.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
-import { StakingPool } from "hooks";
 import { useEffect, useState } from "react";
+
```
[the remark this answers is not in the document]
Comment (dohaki) [reply src/views/Staking/hooks/useStakeFormLogic.ts:2]: That's actually on purpose 😅 I like to separate import blocks based on whether the modules are third-party, globally shared, or local modules... Keeps imports more organized IMO. But yea in this specific file there aren't many import statements.

[the remark this answers is not in the document]
Comment (amateima) [reply src/views/Staking/hooks/useStakeFormLogic.ts:2]: I also like to separate import blocks

**Review by amateima — APPROVED** [review]

**Review by amateima — COMMENTED** [review]
Should I be able to see the new added Pool using the preview link? 'cause I'm not

File: api/_utils.ts [hunk 4]
```diff
@@ -856,7 +856,7 @@ async function getBalancerPoolState(poolTokenAddress: string) {
   const apr = await balancer.pools.apr(pool);
 
   return {
-    estimatedApy: Number(apr.max / 1000).toFixed(2),
+    estimatedApy: Number(apr.max / 10_000).toFixed(2),
```
[the remark this answers is not in the document]
Comment (dohaki) [reply api/_utils.ts:859]: The Balancer SDK returns percentage values as
```
23% (0.23) as 2300
1% (0.01) as 100
```
But we use the decimals presentation, so we need to divide by `10_000`. I will add this as a comment

**Comment by dohaki** [comment]
> Should I be able to see the new added Pool using the preview link? 'cause I'm not

Yes actually you should. But not the liquidity pools but on the staking pool page, i.e. `/rewards` page.

File: src/views/Staking/hooks/useStakeFormLogic.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (james-a-morris) [reply src/views/Staking/hooks/useStakeFormLogic.ts:2]: SGTM!

**Review by james-a-morris — APPROVED** [review]
Looks great, @dohaki

**Pull request merged**